doc/source/whatsnew/v0.17.0.txt

```diff
@@ -45,6 +45,45 @@ Other API Changes
 - Enable writing Excel files in :ref:`memory <_io.excel_writing_buffer>` using StringIO/BytesIO (:issue:`7074`)
 - Enable serialization of lists and dicts to strings in ExcelWriter (:issue:`8188`)
 
+- ``DataFrame.convert_objects`` keyword arguments have been shortened. (:issue:`10265`)
```

Comment:
Can you put everything you added into a separate section above 'Other API Changes' (with the same header level as that one), like "Changes to convert_objects" ?